### PR TITLE
feat(tooling): fix logo/avatar/icon image, move to core and cleanup debris

### LINKED ISSUE #3939: Fix logo/avatar/icon image, move to core and cleanup debris
Refactor everything related to icons, avatars, or whatever they're called, to ensure consistent icons across all the apps, move to core folder and cleanup all debris across the apps

Example:
Image

### PR DESCRIPTION
**Pull request by evavirseda**
# Description of change

Please write a summary of your changes and why you made them.

## Links to any relevant issues

fixes https://github.com/iotaledger/iota/issues/3939

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Documentation Fix

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

File: apps/core/src/hooks/stake/useValidatorInfo.tsx [hunk 1]
```diff
@@ -36,10 +36,10 @@ export function useValidatorInfo({ validatorAddress }: { validatorAddress: strin
         system,
         isPendingValidators,
         errorValidators,
-
         currentEpoch,
         validatorSummary,
         name: validatorSummary?.name || '',
+        imageUrl: validatorSummary?.imageUrl || '',
```
Comment (VmMad) [inline apps/core/src/hooks/stake/useValidatorInfo.tsx]: This was supposed to be made in another issue

File: apps/wallet-dashboard/components/Dialogs/Staking/views/DetailsView.tsx [hunk 2]
```diff
@@ -108,7 +109,7 @@ export function DetailsView({
                     <Card type={CardType.Filled}>
                         <CardImage>
                             <ImageIcon
-                                src={null}
+                                src={imageUrl}
```
Comment (VmMad) [inline apps/wallet-dashboard/components/Dialogs/Staking/views/DetailsView.tsx]: Same here

File: apps/wallet-dashboard/components/Dialogs/Staking/views/Validator.tsx [hunk 3]
```diff
@@ -47,7 +47,7 @@ export function Validator({
     return (
         <Card type={isSelected ? CardType.Filled : CardType.Default} onClick={handleClick}>
             <CardImage>
-                <ImageIcon src={null} label={name} fallback={name} size={ImageIconSize.Large} />
+                <ImageIcon src={imageUrl} label={name} fallback={name} size={ImageIconSize.Large} />
```
Comment (VmMad) [inline apps/wallet-dashboard/components/Dialogs/Staking/views/Validator.tsx]: Same here

**Review by brancoder — CHANGES_REQUESTED** [review]
Looks good just lint issues 🙏

**Review by VmMad — APPROVED** [review]
LGTM

**Review by panteleymonchuk — APPROVED** [review]

**Review by brancoder — APPROVED** [review]
LGTM 🎊

File: apps/core/src/components/icon/ImageIcon.tsx
Comment (brancoder) [inline apps/core/src/components/icon/ImageIcon.tsx:1]: This file has a leftover React import that is not added in this Pr but maybe we can remove it?

**Review by brancoder — CHANGES_REQUESTED** [review]
nit: unused import

**Review by brancoder — APPROVED** [review]
LGTM 🎊

**Pull request merged**